gtsam/hybrid/tests/testHybridBayesNet.cpp

```diff
@@ -248,8 +248,10 @@ TEST(HybridBayesNet, Pruning) {
   logProbability +=
       posterior->at(4)->asDiscrete()->logProbability(hybridValues);
 
+  // Regression
   double density = exp(logProbability);
-  EXPECT_DOUBLES_EQUAL(density, actualTree(discrete_values), 1e-9);
+  EXPECT_DOUBLES_EQUAL(density,
```

Comment:
This was *not* a regression but suddenly there is an arbitrary mult factor here?

Reply:
That is the multiplicative factor for the different normalization constant.